pkg/helmpath/lazypath.go

```diff
@@ -47,3 +48,13 @@ func (l lazypath) configPath(elem ...string) string {
 func (l lazypath) dataPath(elem ...string) string {
 	return l.path(xdg.DataHomeEnvVar, dataHome, filepath.Join(elem...))
 }
+
+// dataPath defines the base directory relative to which user specific data files should be stored.
+func (l lazypath) dataDirsPath(elem ...string) string {
```

Comment:
The spec says that local data dir (XDG_LOCAL_DATA) should be read first, and should be pre-pended to the list of XDG_DATA_DIRS.

Reply:
Sorry... `XDG_DATA_HOME`, not `XDG_LOCAL_DATA`. See https://specifications.freedesktop.org/basedir-spec/basedir-spec-latest.html